src/components/Messages/Messages.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import React from "react";
+import type { Node } from "react";
+
+import ViewWithFooter from "../SharedComponents/ViewWithFooter";
+import MessageList from "./MessageList";
+import useMessages from "./hooks/useMessages";
+
+const Messages = ( ): Node => {
+  const userLogin = { user_login: "nickm01" };
```

Comment:
are lines 11-13 necessary here? I think we can just call `useMessages( )`, which will load all messages 1x when the screen first focuses.